packages/core/test/features/css-contains.spec.ts

```diff
@@ -369,12 +403,54 @@ describe('features/css-contains', () => {
                 `,
             });
         });
-        it('should resolve imported global container name', () => {
+        it('should report redeclare of imported container name', () => {
             const { sheets } = testStylableCore({
-                '/imported.st.css': `
+                '/a.st.css': `
```

Comment:
check double declaration in single file with and without `st-global`

Reply:
good catch!